src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/AbstractSpecialFolderProjectTreePropertiesProvider.cs

```diff
@@ -92,7 +92,7 @@ private void ApplySpecialFolderProperties(IProjectTreeCustomizablePropertyValues
 
             // Avoid overwriting icon if the image provider didn't provide one
             propertyValues.Icon = icon ?? propertyValues.Icon;
-            propertyValues.ExpandedIcon = expandedIcon ?? propertyValues.ExpandedIcon;
+            propertyValues.ExpandedIcon = expandedIcon ?? icon ?? propertyValues.ExpandedIcon;
```

Comment:
This is unneeded that's exactly what CPS will do if unset. Here you are overwriting ExpandIcon if already set.

Reply:
Backed the change out.